src/com/lms/service/BorrowerService.java

```diff
@@ -46,15 +48,13 @@
 	 * the given borrower from the given branch, and the book is not overdue, remove
 	 * the loan from the database and return true. If it is overdue, return false.
 	 *
-	 * <p>TODO: What to do when no matching loan exists?
-	 *
 	 * @param borrower the borrower returning the book
 	 * @param book the book being returned
 	 * @param branch the branch from which it was borrowed
-	 * @param dueDate TODO: document this once author clarifies its purpose
```

Comment:
The changes to the parameter name and return type of this method should be in separate commits from the exception-type changes.